File: templates/web/backend/rust/actix-web/sources/status.rs
use serde::Serialize;

#[derive(Serialize)]
#[allow(dead_code)]
pub struct Status {
    pub service: String,
    pub version: String,
}

#[allow(dead_code)]
pub fn get_status() -> Status {
    Status {
        service: "{{project_name}}".to_string(),
        version: "0.1.0".to_string(),
    }
}

File: templates/web/backend/go/gin/sources/status.go
package main

import "runtime"

type Status struct {
	Service   string `json:"service"`
	Version   string `json:"version"`
	GoVersion string `json:"go_version"`
	Uptime    string `json:"uptime"`
}

func GetStatus() Status {
	return Status{
		Service:   "{{project_name}}",
		Version:   "0.1.0",
		GoVersion: runtime.Version(),
	}
}
